lib/pf/firewallsso/PaloAlto.pm

```diff
@@ -71,7 +71,7 @@ sub action {
         if (
             defined($node_info) &&
             (ref($node_info) eq 'HASH') &&
-            $node_info->{'status'} eq $pf::node::STATUS_REGISTERED &&
+            $node_info->{'status'} eq $pf::node::STATUS_UNREGISTERED &&
```

Comment:
I would say we shouldn't have any assumption on the registered vs unregistered status of the node...

If we send a stop, we send it whether the node is reg or unreg...

Otherwise if a user changes its IP address, the stop won't be sent for his old IP if he is registered